### DISCUSSION TITLE
Optional properties when using strict schema validation

### DISCUSSION THREAD

**Discussion by BenTomsett** [rswag/rswag#640] [General]
When using strict schema validation with rswag, is there a way to mark certain properties as optional or not required? Whilst most of our schemas have required properties which are always returned, some of them aren't, or we have one or the other.
TIA!

**Comment by romanblanco** [comment]
@BenTomsett see https://github.com/rswag/rswag/pull/604#issuecomment-1641941197.

https://github.com/rswag/rswag/pull/675 should make it possible.
